src/commands/features/create.ts

```diff
@@ -21,9 +24,80 @@ export default class CreateFeature extends CreateCommand {
         sdkVisibility: Flags.string({
             description: 'The visibility of the feature for the SDKs'
         }),
+        interactive: Flags.boolean({
+            char: 'i',
+            description: 'Interactive Feature Creation Mode'
+        }),
+    }
+
+    prompts = [namePrompt, keyPrompt, descriptionPrompt]
+
+    quickCreateFeatureParams(answers: Record<string, string>): CreateFeatureParams {
+        return {
+            name: answers.name,
+            key: answers.key,
+            description: answers.description,
+            variables: [ 
+                { 
+                    name: answers.name, 
+                    key: answers.key, 
+                    type: 'Boolean' 
+                } 
+            ],
+            variations: [
+                { 
+                    key: 'variation-on', 
+                    name: 'Variation On', 
+                    variables: { [answers.key]: true }
+                },
+                {
+                    key: 'variation-off',
+                    name: 'Variation Off',
+                    variables: { [answers.key]: false }
+                }
+            ],
+        }
+    }
+    
+    async setupTargetingForEnvironments(featureKey: string) {
+        const environmentKeys = await this.getEnvironmentKeys()
+        Object.values(environmentKeys).forEach(async (environmentKey) => {
+            await updateFeatureConfigForEnvironment(
+                this.authToken, 
+                this.projectKey, 
+                featureKey, 
+                environmentKey, {
+                    targets: [{
+                        distribution: [{
+                            percentage: 1,
+                            _variation: 'variation-on',
+                        }],
+                        audience: {
+                            name: 'All Users',
+                            filters: {
+                                filters: [
+                                    {
+                                        type: 'all'
+                                    }
+                                ],
+                                operator: 'and'
+                            }
+                        }
+                    }],
```

Comment:
it may make sense to pull these default configs into a separate file to reduce clutter in the command